components/user_settings/display/user_settings_theme/premade_theme_chooser/premade_theme_chooser.jsx

```diff
@@ -39,10 +42,19 @@ export default class PremadeThemeChooser extends React.PureComponent {
                             onClick={() => this.props.updateTheme(premadeTheme)}
                         >
                             <label>
-                                <img
-                                    alt={'premade theme ' + k}
-                                    className='img-responsive'
-                                    src={premadeTheme.image}
+                                <ThemeThumbnail
```

Comment:
Huge thumbs up here. I've wanted this for ages like the crappy version Devin and I made for our one hackathon.